discovery-infra/test_infra/helper_classes/config/nodes_config.py

```diff
@@ -1,15 +1,44 @@
 from abc import ABC
-from typing import List, Any
+from typing import List
 
 from dataclasses import dataclass
+from munch import Munch
 
 from .base_config import _BaseConfig
 
 
 @dataclass
-class BaseNodesConfig(_BaseConfig, ABC):
+class BaseTerraformConfig(_BaseConfig, ABC):
     """
-    Define all configurations variables that are needed for Cluster during it's execution
+    Define all configurations variables that are needed for Nodes during it's execution
     All arguments must have default to None with type hint
-    TODO: Placeholder for NodesConfig to configure NodesControllers Will be implemented in the next PR
     """
+    worker_memory: int = None
```

Comment:
Any reason why to set the default value to `None`? If you won't mention it - it would be default anyway.

Reply:
All config values are set to None at the moment. After refactoring `env_variables` and separate defaults from environment variables it might changed